src/main/java/ch/njol/skript/test/runner/StructParse.java

```diff
@@ -100,7 +84,14 @@ public boolean load() {
 		try (RetainingLogHandler handler = SkriptLogger.startRetainingLog()) {
 			String structureSectionNodeKey = ScriptLoader.replaceOptions(structureSectionNodeToParse.getKey());
 			String error = "Can't understand this structure: " + structureSectionNodeKey;
-			Structure.parse(structureSectionNodeKey, structureSectionNodeToParse, error);
+			Structure structure = Structure.parse(structureSectionNodeKey, structureSectionNodeToParse, error);
+
+			if (structure != null) {
+				structure.preLoad();
+				structure.load();
```

Comment:
We need to make sure that preLoad did not return false. postLoad must do the same for load